### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by buggtb**
Updated readme for the charm and added icon.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+To deploy the charm run:
+
+    juju deploy ~spiculecharms/pentaho-bi-server
+
+to make the BI server available you then need to run:
+
+    juju expose pentaho-bi-server
+
+You can then browse to http://ip-address:8080/pentaho to configure the service.
```
Comment (gizmo693) [inline README.md]: Are you configuring the service or should it say server?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+The default username and password is admin/password.
+
+## Scale out Usage
+
+Currently this charm does not support scale out usage.
+
+## Known Limitations and Issues
+
+This charm is currently pretty basic, and doesn't support scaleout/HA usage, this will be resolved in an upcoming release. Did you catch this?
```
Comment (gizmo693) [inline README.md]: Catch what?

File: README.md (hunk 1 above)
Comment (buggtb) [reply README.md]: Hmmm in juju parlance is should probably be Application

**Review by gizmo693 — APPROVED** [review]

**Review by gizmo693 — APPROVED** [review]

**Review by gizmo693 — APPROVED** [review]
I've read through these and it all looks tip top

**Pull request merged**